<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainClass</class>
 <widget class="QMainWindow" name="MainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainClass</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SetRawButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать цвет</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>1</y>
      <width>251</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>251</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>-245</x>
      <y>-69</y>
      <width>1051</width>
      <height>641</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>251</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Ing1_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Лиловый мухомор</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ing2_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Чешуя синей щуки</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ing3_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Лесной мох</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ing4_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Лимонный сок</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ing5_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Печная сажа</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ing6_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Дроблёная соль</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ing1_Count">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string> - 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ing2_Count">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string> - 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ing3_Count">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string> - 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ing4_Count">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string> - 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ing5_Count">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string> - 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ing6_Count">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string> - 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Исходный цвет</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Смешанный цвет</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Чистый цвет</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выберите максимальное</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>количество ингридиентов</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>370</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <zorder>graphicsView_3</zorder>
   <zorder>SetRawButton</zorder>
   <zorder>graphicsView</zorder>
   <zorder>graphicsView_2</zorder>
   <zorder>graphicsView_4</zorder>
   <zorder>Ing1_Label</zorder>
   <zorder>Ing2_Label</zorder>
   <zorder>Ing3_Label</zorder>
   <zorder>Ing4_Label</zorder>
   <zorder>Ing5_Label</zorder>
   <zorder>Ing6_Label</zorder>
   <zorder>Ing1_Count</zorder>
   <zorder>Ing2_Count</zorder>
   <zorder>Ing3_Count</zorder>
   <zorder>Ing4_Count</zorder>
   <zorder>Ing5_Count</zorder>
   <zorder>Ing6_Count</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>spinBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
